InstallHalyard.sh

```diff
@@ -258,12 +260,21 @@ if [ -n "$DOWNLOAD"]; then
   add_apt_repositories
 fi
 
-TEMPDIR=$(mktemp -d installhalyard.XXXX)
-
 echo "$(tput bold)Installing Java 8...$(tput sgr0)"
 
 install_java
 
+apt-get install unzip
```

Comment:
Oops, you really want `-y`. If the user responds with `n` the next call to unzip will fail, and since `-e` is set global to the script, the installation will return 1.